transaction-boundaries/package.json

```diff
@@ -33,6 +33,7 @@
     "grunt-browserify": "^5.0.0",
     "grunt-contrib-connect": "~1.0.2",
     "grunt-contrib-copy": "~1.0.0",
+    "grunt-contrib-jshint": "^2.1.0",
```

Comment:
Why?

Reply:
Have you tried to run this example without this package?

I'm having this error, that's why I inserted this package:
<img width="855" alt="Screenshot 2020-05-28 at 13 31 00" src="https://user-images.githubusercontent.com/15003836/83136289-8192ea00-a0e7-11ea-993c-2ae82741e1fb.png">

Reply:
@philippfromme